src/System.Collections.Concurrent/tests/Performance/Perf.ConcurrentQueue.cs

```diff
@@ -0,0 +1,162 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System.Collections.Concurrent.Tests
+{
+    public class Perf_ConcurrentQueue
+    {
+        [Benchmark(InnerIterationCount = 1_000_000), MeasureGCAllocations]
+        public void Ctor()
+        {
+            foreach (BenchmarkIteration iteration in Benchmark.Iterations)
+            {
+                long innerIters = Benchmark.InnerIterationCount / 10;
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < innerIters; i++)
+                    {
+                        new ConcurrentQueue<int>(); // 1
+                        new ConcurrentQueue<int>(); // 2
+                        new ConcurrentQueue<int>(); // 3
+                        new ConcurrentQueue<int>(); // 4
+                        new ConcurrentQueue<int>(); // 5
+                        new ConcurrentQueue<int>(); // 6
+                        new ConcurrentQueue<int>(); // 7
+                        new ConcurrentQueue<int>(); // 8
+                        new ConcurrentQueue<int>(); // 9
+                        new ConcurrentQueue<int>(); // 10
+                    }
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = 10_000_000), MeasureGCAllocations]
+        public void Enqueue()
+        {
+            foreach (BenchmarkIteration iteration in Benchmark.Iterations)
+            {
+                long size = Benchmark.InnerIterationCount;
+                var q = new ConcurrentQueue<int>();
+
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < size; i++)
+                    {
+                        q.Enqueue(i);
+                    }
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = 1_000_000), MeasureGCAllocations]
+        public void TryDequeue(bool exists)
```

Comment:
This is missing the InlineData arguments for `bool exists`

Reply:
Oops, thanks, will fix.